lib/service/service.go

```diff
@@ -2066,6 +2068,26 @@ func (process *TeleportProcess) initSSH() error {
 	return nil
 }
 
+func compareVersions(ctx context.Context, c *Connector) error {
+	resp, err := c.Client.Ping(ctx)
+	if err != nil {
+		return trace.Wrap(err)
+	}
```

Comment:
I think that the remark in the  https://github.com/gravitational/teleport/pull/9383#discussion_r768717826 comment about reusing Ping response that is already invoked in: 
https://github.com/gravitational/teleport/blob/master/lib/service/connect.go#L55
Instead calling  `c.Client.Ping` twice in `registerWithAuthServer` flow is still valid. 

Can we consider removing this ping call and reusing auth ping response obtained in https://github.com/gravitational/teleport/blob/master/lib/service/connect.go#L55  ?